lib/diagnostic.js

```diff
@@ -47,7 +47,9 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire.bind(alderaan);   // or .call?
```

Comment:
So close! 
`.bind()` binds the context, but doesn't actually call the function like `.call()` does.
You also need a return statement in order for tests to pass.
`return deathStar.fire.bind(alderaan)()`
OR
`return deathStar.fire.call(alderaan)`

I think the second option is better.